pkg/cmd/datamgr/cli/server/server.go

```diff
@@ -316,6 +343,7 @@ func (s *server) runControllers() error {
 		s.dataMover,
 		s.snapManager,
 		os.Getenv("NODE_NAME"),
```

Comment:
It is not part of this change, but it will be highly relevant to get the external data manager working as expected.

My understanding is the external data manager will be expected to run as docker container in a VM deployed via OVF. Do we document the OVF configuration and Dockerfile somewhere? What is the ENV variable, `NODE_NAME`, in the docker container is set to?

the ENV variable, `NODE_NAME`, needs to be unique among all data manager containers (in a scaled-out deployment), as the distributed lock in upload/download controller uses the nodeName(which is set to `os.Getenv("NODE_NAME")` now) as the identity of each lease holder. We can consider container ID (but, it may not be unique across docker hosts) or UUID.

Reply:
Good point. As of now I do not set the NODE_NAME env. I have tested with only a single data manager. We should investigate more on it.
Doc: https://confluence.eng.vmware.com/pages/viewpage.action?pageId=652140821
OVA repo: https://gitlab.eng.vmware.com/core-build/photon-cnsdp

We need to document how this OVA needs to be configured.